lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/StarLasuLWLanguage.kt

```diff
@@ -63,10 +66,12 @@ object StarLasuLWLanguage : Language("com.strumenta.StarLasu") {
             addProperty("position", position, Multiplicity.OPTIONAL)
         }
 
+        addPrimitiveType(TokensList::class)
         ParsingResult = addConcept("ParsingResult").apply {
             addContainment("issues", Issue, Multiplicity.MANY)
             addContainment("root", ASTNode, Multiplicity.OPTIONAL)
```

Comment:
Also here, if we could use property references for names it could be slightly more robust